### DISCUSSION TITLE
Any reason why peer dependancies are locked to React 16?

### DISCUSSION THREAD

**Discussion by jwilliams-met** [remarkjs/remark#991] [General] [answered]
Just reviewing my code base and realise I am getting a warning re: using react-markdown with react 17
`react-markdown@7.0.1" has unmet peer dependency "@types/react@>=16".`
Is there any reason why this hasn't been bumped up to v17?

**Comment by ChristianMurphy** [comment, marked as answer]
It isnt locked, the version is a range, anything in version 16, 17, or 18 is accepted by the range.
React 16 hasn't been dropped from the supported range because there hasn't been a reason to yet.
